src/foam/nanos/auth/Address.js

```diff
@@ -79,7 +82,8 @@ foam.CLASS({
       targetDAOKey: 'countryDAO',
       name: 'countryId',
       of: 'foam.nanos.auth.Country',
-      documentation: 'Country address.',
+      documentation: `A generated name that doesn't contain any special characters which represents
```

Comment:
countryId is a foreign key into the CountryDAO.  By chance the ID matches the human readable ISO country code.